examples/run_largecyc_CPMG.py

```diff
@@ -0,0 +1,203 @@
+"""
+CPMG with large phase cycle
+============================
+
+This script will perform a standard CPMG experiment,
+but will perform a full four-step cycle on the first pulse (to discriminate
+between CP and CPMG), as well as independently cycle the first 180.  The phase
+argument is like this (where Δp₃ is the change from the *second* 180 onwards):
+
+Δp₁ l + (Δp₂)(m) + (Δp₁+Δp₂+Δp₃)(n) = (Δp₁)(l+n) + (Δp₂)(m+n) + (Δp₃)(n)
+
+we will just name l m and n these by the coherence pathways that they label:
+
+:l: ph1
+:m: ph2
+:n: ph_overall
+
+(diff this against `run_CPMG.py`)
+
+In order to form a symmetric echo, a padding time is added before 
+and after your tau through a series of delays.
+If you wish to keep the field as is without adjustment, follow
+the 'py run_CPMG.py' command with 'stayput' (e.g. 'py run_CPMG.py stayput')
+"""
+from pylab import *
+from pyspecdata import *
+import os, sys
+from numpy import *
+import SpinCore_pp
+from SpinCore_pp import prog_plen, get_integer_sampling_intervals, save_data
+from SpinCore_pp.ppg import generic
+from datetime import datetime
+from Instruments.XEPR_eth import xepr
+
+my_exp_type = "ODNP_NMR_comp/CPMG"
+assert os.path.exists(getDATADIR(exp_type=my_exp_type))
+# {{{importing acquisition parameters
+config_dict = SpinCore_pp.configuration("active.ini")
+(
+    nPoints,
+    config_dict["SW_kHz"],
+    config_dict["echo_acq_ms"],
+) = get_integer_sampling_intervals(
+    SW_kHz=config_dict["SW_kHz"],
+    time_per_segment_ms=config_dict["echo_acq_ms"],
+)
+# }}}
+# {{{create filename and save to config file
+config_dict["type"] = "CPMG"
+config_dict["date"] = datetime.now().strftime("%y%m%d")
+config_dict["cpmg_counter"] += 1
+# }}}
+# {{{ command-line option to leave the field untouched (if you set it once, why set it again)
+adjust_field = True
+if len(sys.argv) == 2 and sys.argv[1] == "stayput":
+    adjust_field = False
+# }}}
+input(
+    "I'm assuming that you've tuned your probe to %f since that's what's
```

Comment:
you can't linebreak in the middle of a string like this -- that's why flake is failing right now

Reply:
(there are / should be other flake errors after this, but b/c this one is breaking, that's all it's giving you until you fix it --> you can see the flake errors by clicking for more info where it tells you it fails.)